tcf_website/static/base/bugform.js

```diff
@@ -0,0 +1,31 @@
+function submitForm(){
+  var form = document.getElementById("bugform");
+  var valid = validateForm(form);
+  if (valid === true) {
+    postToDiscord();
+    // reset form after submit
+    $('#bugModal').modal('toggle');
+    form.classList.remove('was-validated');
+    var descriptionField = document.getElementById("descriptionField");
+    descriptionField.value = "";
+  }
+}
+
+function postToDiscord(){
+  var url = $("#urlField").val();
+  var email = $("#emailField").val();
+  var description = $("#descriptionField").val();
+  var data = {
+    "content": "Bug Found! \n**URL:** " + url + "\n**Description**: \n" + description + "\n**Email:** " + email
+  }
+
+  var discordURL = "https://discordapp.com/api/webhooks/767189878223142942/wwBQA0K4VQ0i94ku2os2tYIVpPbDtfeP1i6s5G3CBWSCI7R0t6PbhZxwgJ8Z2yYpyv-q"
+  $.post( discordURL,
+          data,
+          function(){ console.log("posted to discord") });
```

Comment:
Log should be removed if we know the posting works now

Reply:
done